### DISCUSSION TITLE
Potential Build Issues

### DISCUSSION THREAD

**Discussion by ajyano22** [idaholab/moose#16986] [Q&A Getting Started] [answered]
Hey guys, I'm new to moose and linux and am having what I believe are build issues when trying to set up and run moose. I thought I had it installed correctly, but I kept getting a 'command not found' error when trying to run a simple make -j 4 test. going through the troubleshooting guide I ran into other issues, specifically my 'hello world' test also failed. What did I mess up?
![Screenshot_2021-02-12_12-34-05](https://user-images.githubusercontent.com/78992306/107839472-57fb1b00-6d69-11eb-9afc-1eb7eb3e2c76.png)

**Comment by GiudGiud** [comment, marked as answer]
Hello

You need to activate the conda environment every time you open a new console.
So ‘conda activate moose’.

Did you install in /tmp? Or was this for your hello world test?

Best
Guillaume

**Reply by ajyano22** [reply]
Hello,

You're right, that was stupid of me. The hello world test is working now but still not the make tests? The /tmp was for the hello world test. 
![image](https://user-images.githubusercontent.com/78992306/108148882-f69bba80-708e-11eb-80c1-6d20baec732f.png)

**Reply by milljm** [reply]
`make` command not found would indicate either your PATH is not set up correctly, or the necessary development tools on this machine are not installed. I am betting its the former PATH issue. Most Linux machines (if not all of them) come with `make`.

Can you verify what PATH is set to for us?

```bash
echo $PATH
```

**Reply by ajyano22** [reply]
Here's that path 
![image](https://user-images.githubusercontent.com/78992306/108233404-fa682500-7100-11eb-9938-f374a5344dcf.png)

**Reply by milljm** [reply]
Please refrain from using pictures of text. It prevents these forums from being searchable for others looking for answers. Simple copy paste of text suffices.

The path looks correct. Can you still not run `make`:

```bash
make --version
```

**Reply by ajyano22** [reply]
Good to know, thanks for the tip. 
It's still not running the make command.
(moose) [aliyay@aliya-hpspectrex360convertible15ch0xx ~]$ make --version
bash: make: command not found
(moose) [aliyay@aliya-hpspectrex360convertible15ch0xx ~]$ cd ~/projects/moose/test
(moose) [aliyay@aliya-hpspectrex360convertible15ch0xx test]$ make --version
bash: make: command not found

**Reply by milljm** [reply]
What flavor of Linux are you using?

**Reply by ajyano22** [reply]
Manjaro

**Reply by milljm** [reply]
Looks like Manjaro is an Arch based distribution. You are going to need to install the development tools for that platform first. The first link I found is the following:

  https://ostechnix.com/install-development-tools-linux/

Strangely, I don't easily find a "HowTo" page for Manjaro and software development. This page may also help, but it looks very Arch-linux centric: https://bbs.archlinux.org/viewtopic.php?id=61739

**Reply by ajyano22** [reply]
It's working now! That was it. Thank you for your help!
